### PR TITLE
first draft of 01-Howto-github.md

### PR DESCRIPTION
**Pull request by vpbrendel**
This document is intended to explain and review our basic use of our github repositories.

### CODE REVIEW

File: doc/01-Howto-github.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# 01-Howto-githup
```
Comment (standage) [inline doc/01-Howto-github.md]: Minor typo here: githup --> github.

File: doc/01-Howto-github.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Syncing your __origin__ fork with the __upstream__ repository
+
+To complete the creative triangle, we need to be able to bring our own __origin__ repository up to date with respect to other changes that may have been accepted into the __upstream__ repository - after all we want to work as a team, and thus we won't be the only ones contributing.
+To do this, you will need to first update your __sandbox__, then push those updates to your __origin__ repository.
+Continuing with our example:
+
+```bash
+[]git remote add upstream https://github.com/brendelgroup/bghandbook
+[]]git remote -v
```
Comment (standage) [inline doc/01-Howto-github.md]: Typo: extra bracket.

**Comment by standage** [comment]
Looks great! If you fix those typos and submit another commit to your master branch, I'll go ahead and accept the pull request.

**Comment by vpbrendel** [comment]
Typos fixed.
Volker

On 1/15/2016 2:20 PM, Daniel Standage wrote:
>
> Looks great! If you fix those typos and submit another commit to your 
> master branch, I'll go ahead and accept the pull request.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BrendelGroup/bghandbook/pull/11#issuecomment-172058735>.
>

-- 
Volker Brendel
Professor of Biology and Computer Science
Indiana University
Department of Biology&  School of Informatics and Computing
212 South Hawthorne Drive
Simon Hall 205C
Bloomington, IN 47405

Tel.: (812) 855-7074
http://brendelgroup.org/

**Pull request merged**